### ISSUE TITLE
Add remote consistency check capabilities to reference validation programs

### ISSUE THREAD

**Issue by jmfernandez** [inab/extended-json-schema-validators#1]
As the benchmarking API is approaching a beta state, sooner than later benchmarking communities are going to be involved in the data uploading process. So, when the reference validation programs apply the consistency checks, like primary key uniqueness or foreign keys not dangled, they should also ask the benchmarking API about this.

**Comment by javi-gv94** [comment]
+1 for this! Currently starting to work in a 'tool' to upload data automatically from a VRE workflow to OEB Mongo. For now I am only able to validate local objects independently, but would be extremely useful to add capability to validate foreign keys against remote OEB data.

**Comment by jmfernandez** [comment]
I apologize in advance, I have realized I left behind this issue, related to the python implementation of the extended JSON validator.

As the development around this validator was spread through forks over several repositories, I was working in October on the reunification of all the improvements in a common place (https://github.com/inab/python-extended-json-schema-validator/), and publishing the developments in PyPI (https://pypi.org/project/extended-json-schema-validator/).

I have re-read the issue, and the enhancement was implemented in a different repo, and inherited by the "re-unified" code. Through command line it is possible to tell a configuration file which can set up remote endpoints where to fetch already valid public identifiers related to the different JSON Schemas used to validate the contents. It is described at https://github.com/inab/python-extended-json-schema-validator/blob/main/README-extensions.md , under "Primary key values check".

Unless anyone has hesitations, I'm closing this issue.
